util/topgen/rust.py

```diff
@@ -49,7 +49,7 @@ def size_words_name(self, short=False):
 
 
 class RustEnum(object):
-    def __init__(self, top_name, name, repr_type=None, derive_list=["Copy", "Clone"]):
+    def __init__(self, top_name, name, repr_type=None, derive_list=["Copy", "Clone","PartialEq", "Eq"]):
```

Comment:
You'll need to run //util/lintpy.py to correct this.

I think the linter is compaining about both the line length and a missing space after the comma.